rhobs/alerting/data_plane/prometheus.gitops_deployment_alerts.yaml

```
@@ -0,0 +1,32 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: rhtap-gitops-deployment-alerting
+  labels:
+    tenant: rhtap
+spec:
+  groups:
+  - name: gitops_deployment_alerts
+    interval: 1m
+    rules:
+    - alert: GitOpsDeploymentHealthErrors
+      expr: |
+        (
+          sum(kube_deployment_status_replicas_ready{namespace="gitops-service-argocd"})
+          by (source_cluster)
+        )
+        /
+        (
+          sum(kube_deployment_spec_replicas{namespace="gitops-service-argocd"})
+          by (source_cluster)
+        ) < 0.95
+      for: 5m
+      labels:
+        severity: warning
+        slo: true
```

```suggestion
        slo: "true"
```

We recently discovered a bug with boolean label values (see #138). Please update all the `slo` labels across your PRs. Sorry for the inconvenience!

---

NP, I updated it, PTAL.